TESTS/mbed_drivers/flashiap/main.cpp

```diff
@@ -22,6 +22,7 @@
 #include "utest/utest.h"
 #include "unity/unity.h"
 #include "greentea-client/test_env.h"
+#include <algorithm>
```

Comment:
What exactly is this importing? This is the first time I've ever seen a system file simply called "algorithm".

Reply:
It is actually used quite a lot. It imports the std::min and std::max functions.